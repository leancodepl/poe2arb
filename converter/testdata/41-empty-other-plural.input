[
    
    {
        "term": "testPlural",
        "definition": {
            "one": "",
            "few": "",
            "many": "",
            "other": ""
        },
        "context": "",
        "term_plural": "plural",
        "reference": "",
        "comment": ""
    }
]
