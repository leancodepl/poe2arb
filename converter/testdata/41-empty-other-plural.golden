{
    "@@locale": "en",
    "testPlural": "",
    "@testPlural": {
        "placeholders": {
            "count": {
                "type": "num",
                "format": "decimalPattern"
            }
        }
    }
}
